ui/app/templates/components/console/log-error-with-html.hbs

```diff
@@ -1,5 +1,5 @@
 {{! template-lint-disable no-triple-curlies}}
 <div class="console-ui-alert has-text-danger">
-  {{i-con glyph="close-circled" aria-hidden="true" size=12}}
+  <Icon @glyph="cancel-circle-fill" aria-hidden="true" />
   <pre>{{{content}}}</pre>
```

Comment:
Yeah, since this is possibly inserting HTML, we don't want to escape at the template layer. I think the only time this is really used is in the control group flow - if you hit a Control Group, we log the token and a link to the lookup page right in the console.